code/datums/greyscale/config_types/greyscale_configs/greyscale_clothes.dm

```diff
@@ -628,3 +628,57 @@
 /datum/greyscale_config/witch_hat/worn
 	name = "Witch Hat (Worn)"
 	icon_file = 'icons/mob/clothing/head/wizard.dmi'
+
+/datum/greyscale_config/wellworn_shirt
+	name = "Well-Worn Shirt"
+	icon_file = 'icons/obj/clothing/suits/costume.dmi'
+	json_config = 'code/datums/greyscale/json_configs/wellworn_shirt.json'
+
+/datum/greyscale_config/wellworn_shirt/worn
+	name = "Well-Worn Shirt (Worn)"
+	icon_file = 'icons/mob/clothing/suits/costume.dmi'
+
+/datum/greyscale_config/wellworn_shirt_graphic
+	name = "Well-Worn Shirt (Graphic)"
+	icon_file = 'icons/obj/clothing/suits/costume.dmi'
+	json_config = 'code/datums/greyscale/json_configs/wellworn_shirt_graphic.json'
+
+/datum/greyscale_config/wellworn_shirt_graphic/worn
+	name = "Well-Worn Shirt (Graphic)(Worn)"
+	icon_file = 'icons/mob/clothing/suits/costume.dmi'
+
```

Comment:
```suggestion
/datum/greyscale_config/wellworn_shirt/graphic
	name = "Well-Worn Shirt (Graphic)"
	json_config = 'code/datums/greyscale/json_configs/wellworn_shirt_graphic.json'

/datum/greyscale_config/wellworn_shirt/worn/graphic
	name = "Well-Worn Shirt (Graphic)(Worn)"
	
```
These (and all the others below ofc) could be subtypes to inherit more info without repeating it, but it would mean a bit of re-organizing on your part...